profiles/kentik_snmp/fortinet/fortinet-fortigate.yml

```diff
@@ -375,3 +375,27 @@ metrics:
       - column:
           name: fgHaStatsHostname
           OID: 1.3.6.1.4.1.12356.101.13.2.1.1.11
+
+  - MIB: FORTINET-FORTIGATE-MIB
+    table:
+      name: fgLinkMonitorTable
+      OID: 1.3.6.1.4.1.12356.101.4.8.2
+    symbols:
+      - name: fgLinkMonitorState
+        OID: 1.3.6.1.4.1.12356.101.4.8.2.1.3
+        enum:
+          alive: 0
+          dead: 1
+      - name: fgLinkMonitorLatency
+        OID: 1.3.6.1.4.1.12356.101.4.8.2.1.4
+      - name: fgLinkMonitorJitter
+        OID: 1.3.6.1.4.1.12356.101.4.8.2.1.5
+      - name: fgLinkMonitorPacketLoss
+        OID: 1.3.6.1.4.1.12356.101.4.8.2.1.8
+    metric_tags:
+      - column:
+          name: fgLinkMonitorID
+          OID: 1.3.6.1.4.1.12356.101.4.8.2.1.1
```

Comment:
@al3kh - this should already be covered natively and stored on the dimensional metrics as the `Index` attribute. It shouldn't be needed here.

curious though; does the [fgLinkMonitorVdom](https://www.circitor.fr/Mibs/Html/F/FORTINET-FORTIGATE-MIB.php#fgLinkMonitorVdom) name not add value for your use cases? I don't know much about the VDOMs in Fortigate, but thought I'd ask. 😄

Reply:
@thezackm yeah i agree it would so removed the ID. thankyou